app/components/swarmVisualizer/swarmVisualizer.html

```diff
@@ -73,13 +73,17 @@
                 </div>
               </div>
               <div>{{ node.Role }}</div>
+              <div>CPU: {{ node.CPUs / 1000000000 }}</div>
+              <div>Memory: {{ node.Memory|humansize: 2 }}</div>
             </div>
             <div class="tasks">
               <div class="task task_{{ task.Status.State | visualizerTask }}" ng-repeat="task in node.Tasks | filter: (state.DisplayOnlyRunningTasks || '') && { Status: { State: 'running' } }">
                 <div class="service_name">{{ task.ServiceName }}</div>
                 <div>Image: {{ task.Spec.ContainerSpec.Image | hideshasum }}</div>
                 <div>Status: {{ task.Status.State }}</div>
                 <div>Update: {{ task.Updated | getisodate }}</div>
+                <div ng-if="task.Spec.Resources.Limits.MemoryBytes">Memory limit: {{ task.Spec.Resources.Limits.MemoryBytes | humansize: 2 }}</div>
```

Comment:
> Note that 128M in Docker means 128MiB which is the equivalent to 134.22 MB.

In order to be coherent within the UI, I'd like to display this information as 128MB. To do so, you just need to change the `humansize` fitler to expose a new parameter called `base` (here we need to use base 2).